src/smc-webapp/project.coffee

```diff
@@ -172,6 +174,14 @@ class ProjectPage
                         box.hide()
 
 
+    requests_to_join_project_alert: (project_id) =>
+        if Object.keys(redux.getStore('projects').get_project(project_id).invite_requests).length > 0
```

Comment:
There is no guarantee that `redux.getStore('projects').get_project(project_id)` isn't undefined when this is called.  Maybe the projects hasn't completely loaded yet?  I'm hitting this.